opencog/benchmark/atomspace_bm.cc

```diff
@@ -25,6 +27,8 @@ int main(int argc, char** argv)
      "          \t(default: 100000)\n"
      "-r <int>  \tLooping count; how many times a python/scheme operation is looped\n"
      "          \t(default: 1)\n"
+     "-R <int>  \tUse specific randomseed; useful for benchmark comparisons\n"
+     "          \t(default: time(NULL))\n"
```

Comment:
I don't mind the -R flag, but the default is wrong.  Is there any reason to beleive that the results depend on the random number?